go.mod

```diff
@@ -10,11 +10,11 @@ require (
 	github.com/containernetworking/cni v0.7.2-0.20200304161608-4fae32b84921
 	github.com/containernetworking/plugins v0.8.5
 	github.com/containers/buildah v1.14.5
-	github.com/containers/common v0.6.1
+	github.com/containers/common v1.0.0
```

Comment:
0.8.0 is the latest here isn't it?

Reply:
@vrothberg Something strange is going on in vendor.  For some reason make vendor is replacing v0.8.0 with v1.0.0 and then causing everything to blow up.

Reply:
I think we need to update buildah, storage and common together.  Hopefully this passes.
@TomSweeneyRedHat You never pushed a tag for 1.14.6 in buildah, so I could vendor this.

Reply:
@rhatdan just tagged a release.